src/Crocodealer/Command/Sync.hs

```diff
@@ -27,16 +26,10 @@ sync userName templateRepoName fileName repos =
                         let fileDiff = calculateDiff f r
                         printFileDiff userName repo fileDiff
 
-fetchGitHubFile :: UserName -> RepoName -> FileName -> IO (Maybe ByteString)
+fetchGitHubFile :: UserName -> RepoName -> FileName -> IO (Maybe Text)
 fetchGitHubFile userName repoName fileName = do
     let url = "https://raw.githubusercontent.com/" <> unUserName userName <> "/" <> unRepoName repoName <> "/master/" <> unFileName fileName
-    req <- parseRequest $ toString url
-    resp <- httpBS req
-
-    pure $ case getResponseStatus resp of
-        Status 200 _ -> Just $ getResponseBody resp
-        _            -> Nothing
-
+    (sequence . Just) ("curl" $| ["-s", "--fail", url]) $? pure Nothing
```

Comment:
What benefits `--fail` gives us? Maybe it's okay to see the failure reason?

Reply:
I would also add, that this can be written slightly better like this:

```suggestion
    (Just <$> "curl" $| ["-s", "--fail", url]) $? pure Nothing
```

Precedence of operators in `shellmet` was specifically chosen that way to support nicer interaction with functors.

Reply:
@vrom911 I added --fail because it fails the command when the file isn't found. Otherwise the command itself doesn't fail, but returns a 404 Not Found string.

Reply:
You are saying that it puts this 404 string as the successful output? That doesn't sounds right :thinking:

Reply:
Here's a screenshot ( this is on MacOS, I saw the same behaviour on Linux as well ): 

![image](https://user-images.githubusercontent.com/15860618/75464110-651fce00-598f-11ea-82e1-4fd2ac57ceac.png)

Reply:
wow :scream: Thanks for catching that!